hd/etc/menubar.txt

```diff
@@ -342,11 +342,8 @@ fr: Sélectionnez les modules en cliquant sur le bouton correspondant (max 15).
                 <a class="dropdown-item" href="%prefix;%access;&m=D"><span class="fa fa-cog fa-fw mr-2"></span>[*descendants]</a>
                 <div class="dropdown-divider"></div>
                 <a class="dropdown-item" href="%prefix;%access;&m=D&t=V&%if;(e.v!="")v=%e.v;%else;v=3%end;&sp=1&ma=1"%if;(not has_parents) %laY;%end;><span class="fa fa-sitemap fa-fw mr-2"></span>[*descendants tree]</a>
-                <a class="dropdown-item" href="%prefix;%access;&m=D&t=TV&v=4&sp=1&ma=1"%if;(not has_parents) %laY;%end;><span class="fa fa-sitemap fa-fw mr-2"></span>[*compact descendants tree]</a>
-                %(
-                <a class="dropdown-item" href="%prefix;%access;&m=D&t=T&v=4"%if;(not has_parents) %laY;%end;><span class="fa fa-sitemap fa-fw mr-2"></span>[*descendants tree]</a>
-                %)
-                <a class="dropdown-item" href="%prefix;%access;&m=D&t=D&v=%max_desc_level;"><span class="fa fa-code-branch fa-rotate-90 fa-fw mr-2"></span>[*descendant tree view]</a>
+                <a class="dropdown-item" href="%prefix;%access;&m=D&t=TV&%if;(e.v!="")v=%e.v;%else;v=3%end;&sp=1&ma=1"%if;(not has_parents) %laY;%end;><span class="fa fa-sitemap fa-fw mr-2"></span>[*compact descendants tree]</a>
```

Comment:
@a2line , I strongly dislike the fact you pulled commit I signed with changes (in summary and code changes)  without any previous discussions in related PR (https://github.com/hgouraud/geneweb/pull/7).  Not a good practice.

Reply:
https://fr.wiktionary.org/wiki/psychorigidité 2.0